lib/ansible/modules/cloud/azure/azure_rm_functionapp.py

```diff
@@ -280,10 +280,18 @@ def aggregated_app_settings(self):
 
         function_app_settings = self.necessary_functionapp_settings()
         for app_setting_key in self.app_settings:
-            function_app_settings.append(NameValuePair(
-                name=app_setting_key,
-                value=self.app_settings[app_setting_key]
-            ))
+            found_setting = None
+            for s in function_app_settings:
+                if s.name == app_setting_key:
+                    found_setting = s
+                    break
```

Comment:
what if multiple existing keys get updated in self.app_settings? break here will only update one

Reply:
this will never happen, function_app_settings come from the function above and are fixed set.

Reply:
even no duplicate, can i update 2 settings at  once? eg. FUNCTIONS_EXTENSION_VERSION, MYSETTINGS' 2 keys

Reply:
yes, you can